packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -82,6 +86,19 @@ const Trade = observer(() => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    React.useEffect(() => {
+        const { has_error, error_field, error_code, message } = proposal_info?.[trade_type_tab] || {};
+
+        if (has_error && !error_field && error_code === 'ContractBuyValidationError') {
```

Comment:
Is it the same error, that was handled in services-error-snackbar.tsx in [this PR](https://github.com/deriv-com/deriv-app/pull/17344/files#diff-642f163cbf10ee0237a7b2af52bb6ea55d71dce26d1b0a9d6d9130cb639eb910)?
I also did some check like `has_error && !error_field`

Reply:
Could you please check?

Reply:
@kate-deriv Seems like your PR is fixing this error as well, but the errors are different though, this one is not a service error. There are many errors that can come without a specific field.

Reply:
What do you think we should do? I can remove this logic but also it doesn't make sense to show this contract specific message inside service errors component?

Reply:
I see that you are excluding contract errors from your logic. Is it what you were trying to do there? If yes, maybe you could add error_code in the condition to exclude this error from services-error?

Reply:
I was trying to create a component for handling only service errors. But it turns out, that one BO error (when symbol is unavailable) is coming  not inside of service error, but as a part of proposal_info. That's why I added this error in my component too.
Let me check the error code of that error.

Reply:
Okay, it actually has ContractBuyValidationError error code

Reply:
What do you think if we create a separate component for handling contract errors? Which are coming without error field

Reply:
Okay, I guess you can just delete it. After my PR will me merged, it should work

Reply:
Yeah, I'll just delete it or it will cause showing the same snackbar twice on production

Reply:
I removed this